SimTKcommon/Scalar/include/SimTKcommon/internal/NTraits.h

```diff
@@ -1073,9 +1073,12 @@ template<> struct NTraits<R>::Result<conjugate<double> > \
   {typedef conjugate<Widest<R,double>::Type> Mul;typedef Mul Dvd;typedef Mul Add;typedef Mul Sub;}; \
 template<> struct NTraits<R>::Result<conjugate<long double> > \
   {typedef conjugate<Widest<R,long double>::Type> Mul;typedef Mul Dvd;typedef Mul Add;typedef Mul Sub;}
+#pragma clang diagnostic push
```

Comment:
I think it is necessary to add additional pragmas to first detect clang. Otherwise, other compilers will give warnings.